src/utils/hooks/useFeaturedBanners.js

```diff
@@ -1,48 +0,0 @@
-import { useState, useEffect } from 'react';
```

Comment:
This file could be useful in the future, if you could change it and make it so you implement here the logic for the banners and all the other data would be better, as a hook